msteams-platform/tabs/how-to/create-channel-group-tab.md

```diff
@@ -562,42 +596,75 @@ Ensure that you keep the command prompt with ngrok running and make a note of th
+    Within **Tab.cshtml** the application presents the user with two option buttons for displaying the tab with either a red or gray icon. Choosing the **Select Gray** or **Select Red** button triggers `saveGray()` or `saveRed()`, respectively, sets `pages.config.setValidityState(true)`, and enables the **Save** button on the configuration page. This code lets Teams know that you have completed the configuration requirements and the installation can proceed. The parameters of `pages.config.setConfig` are set. Finally, `saveEvent.notifySuccess()` is called to indicate that the content URL has been successfully resolved. 
 
 1. Update the `websiteUrl` and `contentUrl` values in each function with the HTTPS ngrok URL to your tab.
 
     Your code should now include the following with **y8rCgT2b** replaced with your ngrok URL:
 
-    ```javascript
-
-        let saveGray = () => {
-            microsoftTeams.settings.registerOnSaveHandler(function (saveEvent) {
-                microsoftTeams.settings.setSettings({
-                    websiteUrl: `https://y8rCgT2b.ngrok.io`,
-                    contentUrl: `https://y8rCgT2b.ngrok.io/gray/`,
-                    entityId: "grayIconTab",
-                    suggestedDisplayName: "MyNewTab"
-                });
-                saveEvent.notifySuccess();
+# [TeamsJS v2](#tab/teamsjs-v2)
+
+```javascript
+
+    let saveGray = () => {
+        pages.config.registerOnSaveHandler(function (saveEvent) {
+            pages.config.setConfig({
+                websiteUrl: `https://y8rCgT2b.ngrok.io`,
+                contentUrl: `https://y8rCgT2b.ngrok.io/gray/`,
+                entityId: "grayIconTab",
+                suggestedDisplayName: "MyNewTab"
             });
-        }
+            saveEvent.notifySuccess();
+        });
+    }
     
-        let saveRed = () => {
-            microsoftTeams.settings.registerOnSaveHandler(function (saveEvent) {
-                microsoftTeams.settings.setSettings({
-                    websiteUrl: `https://y8rCgT2b.ngrok.io`,
-                    contentUrl: `https://y8rCgT2b.ngrok.io/red/`,
-                    entityId: "redIconTab",
-                    suggestedDisplayName: "MyNewTab"
-                });
-                saveEvent.notifySuccess();
+    let saveRed = () => {
+        microsoftTeams.settings.registerOnSaveHandler(function (saveEvent) {
```

Comment:
This should probably be pages.config instead of microsoftTeams.settings

Reply:
Thanks. Should be corrected in latest commit: https://github.com/MicrosoftDocs/msteams-docs/pull/5884/commits/b4a65c5b699ccffc3bc647daaff38916f649c46d